.github/workflows/check-csv-ordering.yaml

```diff
@@ -0,0 +1,28 @@
+name: Check CSV order
+
+on:
+
+  push:
+    branches:
+      - ci-check-csv-ordering
+
+  pull_request:
```

Comment:
Not sure if this is correct and how that interacts with the source branch in the PR being on a fork.